src/constants/config.constants.ts

```
@@ -93,9 +93,13 @@ export const FetchAllTvShowsSlateTime = 1 * 60 * 1000; //1 min
 
 //Tv details slate time
 export const FetchSingleTvDetailsStaleTime = 1 * 60 * 1000; //1 min
+export const FetchSeasonDetailsStaleTime = 1 * 60 * 1000; //1 min
 
 export const MEDIA_STATUS = ['released', 'upcoming'];
 
+//season status
+export const SEASON_STATUS = ['upcoming', 'on-going', , 'ended'];
```

Biome (2.1.2)

[error] This array contains an empty slots..

The presences of empty slots may cause incorrect information and might be a typo.
Unsafe fix: Replace hole with undefined


(lint/suspicious/noSparseArray)